app/javascript/Components/Result/submission_selector.jsx

```diff
@@ -174,10 +176,21 @@ export class SubmissionSelector extends React.Component {
   }
 
   render() {
-    if (this.props.role === "Student" && !this.props.is_reviewer) {
+    if (this.context.role === "Student" && !this.context.is_reviewer) {
       return "";
     }
 
+    const progressBarWidth =
```

Comment:
You can delete this block of code (I removed it in my enzyme PR, it was unused).